src/forms/forms.service.ts

```diff
@@ -35,18 +39,72 @@ export const formSelect = {
                     },
                 },
             },
+            subQuestions: {
+                select: {
+                    id: true,
+                    order: true,
+                    inputType: {
+                        select: {
+                            id: true,
+                            name: true,
+                        },
+                    },
+                    text: true,
+                    description: true,
+                    answerRequired: true,
+                    multipleAllowed: true,
+                    optionGroup: {
+                        select: {
+                            optionChoices: {
+                                select: {
+                                    id: true,
+                                    text: true,
+                                },
+                            },
+                        },
+                    },
+                    createdAt: true,
+                    updatedAt: true,
+                },
+            },
+            createdAt: true,
+            updatedAt: true,
         },
     },
-};
+} as Prisma.FormSelect;
 
 @Injectable()
 export class FormsService {
     constructor(private prisma: PrismaService) {}
 
-    getAllForms() {
-        return this.prisma.form.findMany({
+    async getAllForms() {
+        const data = await this.prisma.form.findMany({
```

Comment:
should we use something more descriptive than `data`?

Reply:
I can switch it to: forms?